docs/manual/mod/mod_dav.xml

```diff
@@ -194,6 +194,36 @@ Alias "/php-source" "/home/gstein/php_files"
 </usage>
 </directivesynopsis>
 
+<directivesynopsis>
+<name>DavBasePath</name>
+<description>Configure repository root path</description>
+<syntax>DavBasePath <var>root-path</var></syntax>
+<default>DavBasePath /repos/</default>
```

Comment:
Is "/repos/" as the default value really meaningful?

It is not hard-coded in the module and does not fit well with the comment "Take the repos root from DAVBasePath if configured, **else the path of the enclosing section**" in the [cb4eb29](https://github.com/apache/httpd/pull/377/commits/cb4eb2940eaab5103d9055e2fe25e5a3e717a132)

Reply:
Nope, definitely wouldn't be... thanks for reviewing.